README.md

```diff
@@ -5,31 +5,31 @@
 
 # Stacks
 
 This repository holds the default set of stacks available from Appsody. Each stack is pre-configured with popular open source technologies to enable rapid development of quality microservice-based applications. Stacks include a base container image and project templates which act as a starting point for your application development.
 
 Stacks allow for rapid development whilst giving the stack developer the ability to control the overall applications that are created from it.
 
-To find out more about Appsody check out [appsody.dev](https://appsody.dev).
+Click here to find out more about [Appsody stacks](https://appsody.dev/docs/stacks/stacks-overview).
 
 ## Repository Structure
 Stacks are categorized as either stable, incubator or experimental depending on the content of the stack.
 
-- `stable/`: Stable stacks meet this set of [technical requirements](https://github.com/appsody/stacks/blob/master/TECHNICAL_REQUIREMENTS.md).
+- `stable/`: Stable stacks are production-ready.
 
-- `incubator/:` The stacks in the incubator folder are actively being worked on to satisfy the stable criteria.
+- `incubator/`: Incubator stacks are actively being worked on to satisfy the stable criteria.
 
-- `experimental/:` Experimental stacks are not being actively been worked on and may not fulfill the requirements of an Appsody stack. These can be used for trying out specific capabilities or proof of concept work.
+- `experimental/`: Experimental stacks are not being actively been worked on and may not fulfill the requirements of an Appsody stack. These can be used for trying out specific capabilities or proof of concept work.
 
-Click here to find out more about [Appsody stacks](https://github.com/appsody/website/blob/master/content/docs/stacks/stacks-overview.md).
+Click here to find out more about [Appsody stack stability levels](https://appsody.dev/docs/stacks/stacks-overview#stack-stability-levels).
 
 ## Creating a Stack
 We are actively working to create new stacks so that more people can adopt Appsody. If you find that none of the existing stacks meet your needs please reach out to us on the [Appsody Slack](https://appsody-slack.eu-gb.mybluemix.net/) or create a new GitHub issue to track the discussion.
 
-We always welcome any contributions. If you wanted to create your own stack for a framework or language that we do not currently support, please review the [contributing guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md) and follow the steps outlined in [creating a stack](https://github.com/appsody/website/blob/master/content/docs/stacks/create.md).
+We always welcome any contributions. If you wanted to create your own stack for a framework or language that we do not currently support, please review the [contributing guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md) and follow the steps outlined in [developing a stack](https://appsody.dev/docs/stacks/develop).
```

Comment:
How about streamlining it to: "To create your own stack for a framework or language that is not currently supported, review the [contributing guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md) and follow the steps that are outlined in [developing a stack](https://appsody.dev/docs/stacks/develop)."